readme.md

```diff
@@ -66,6 +69,18 @@ To build and run the example using Bazel, from the root of the repo:
 bazel run examples:septentrio_example -- --help
 ```
 
+#### Cross-Compiling With Bazel ####
+
+The Bazel build flow supports cross-compilation for 32- and 64-bit ARM architectures. To build for either architecture,
+specify the `--config` argument to Bazel with one of the following values:
+- `armv7hf` - 32-bit ARM v7
+- `aarch64` - 64-bit ARM
```

Comment:
I think since we've seen so many people ask about the Nvidia PX2/TX2/Jetson/Pegasus we should clarify aarch64 if the one you want for those platforms

Reply:
Added examples for each.